ntoskrnl/kd/kdio.c

```diff
@@ -412,7 +412,7 @@ KdpScreenPrint(LPSTR Message,
     KIRQL OldIrql;
     PCHAR pch = (PCHAR) Message;
 
-    while (*pch)
+    while (pch < Message + Length && *pch)
```

Comment:
Code style: use `!= '\0'` everywhere/nowhere?

Reply:
I did not touch that part of the condition and I don't intend to.